hermes-frontend/src/main/java/pl/allegro/tech/hermes/frontend/producer/BrokerMessagesProducingResults.java

```diff
@@ -0,0 +1,25 @@
+package pl.allegro.tech.hermes.frontend.producer;
+
+import java.util.Collection;
+
+public final class BrokerMessagesProducingResults {
```

Comment:
Maybe we should name this class more clearly, it can be easily mixed with `BrokerMessagesProducingResult`. You ould name it just like `BrokerMessagesProducingResultsBatch` OSLT :)